static/templates/settings/admin_user_list.hbs

```diff
@@ -1,8 +1,17 @@
 <tr class="user_row{{#unless is_active}} deactivated_user{{/unless}}" data-user-id="{{user_id}}">
+    {{#if is_bot}}
+    <td>
+        <span class="user_name">
+            <a data-user-id="{{user_id}}" class="view_user_profile" tabindex="0">{{full_name}}</a>
```

Comment:
This branch should also have the deactivated user icon. Maybe we can wrap `<a>` inside the if-else statement instead.

Reply:
@PIG208 made the suggested changes, could you please review it again and suggest if any changes are required, Thanks.